src/common/components/faq-category/index.scss

```diff
@@ -3,6 +3,23 @@
     background: $dark;
   }
 }
+
```

Comment:
@MTSaeed You are again modifying generic/common class name out in open. It will affect or change style of other components that use same component. Wrap them with component based parent class name so it doesn't break other same named class styles.

Reply:
@feruzm  Updated.